src/Paket.Core/Paket.Core.fsproj

```diff
@@ -63,6 +63,11 @@
         <FSharpTargetsPath>$(MSBuildProgramFiles32)\Microsoft SDKs\F#\4.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
       </PropertyGroup>
     </When>
+    <When Condition="$(TargetFSharpCoreVersion) &gt;= 4.4.1.0 AND $(TargetFSharpCoreVersion) &lt; 4.4.1.0 ">
```

Comment:
These kinds of conditions don't work with xbuild, it's ridiculous that they don't and I'd rather write them this way too, but they end up causing more headaches than they're worth.